frontend/src/app/commonComponents/Header.tsx

```diff
@@ -251,7 +259,7 @@ const Header = () => {
             </li>
             <li className="usa-nav__primary-item">
               <NavLink
-                to={`/settings`}
+                to={`/facility/${facility.id}/settings`}
```

Comment:
is there an (easy) better solution to this?

Reply:
@timothybest-usds removed the facility id from the path recently.

Reply:
hmm, well the associated issue says to add it back in, but it seems like an alternate solution is in the works: 
https://github.com/CDCgov/prime-simplereport/issues/236

Reply:
This is the correct solution. I overlooked URL pathing when implementing facilities